addons/uh60_weapons/config/cfgWeapons.hpp

```diff
@@ -39,10 +39,31 @@ class cfgWeapons {
           };
       };
     };
+  /*
   class RocketPods;
   class missiles_DAR: RocketPods {
     magazines[] += {"VTX_PylonRack_M261_M229"};
   };
+  */
+  class missiles_DAR{
```

Comment:
Need to inherit `class missiles_DAR: RocketPods {`
else you get `No entry 'bin\config.bin/CfgWeapons/missiles_DAR.scope'.`

Reply:
Addressed in https://github.com/Project-Hatchet/H-60/pull/285/commits/bce1089aa505e96c3e4fdccea86d6442fce648b3

I didn't see this error on my end. Were you seeing this in the dev version or ARMA or is there something else I need to switch on to see this?

Thanks!